### PR TITLE
Add recipe for R 3.2.0

### PR DESCRIPTION
**Pull request by davebx**
Addresses #11

### CODE REVIEW

File: R/build.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+if [ -z $3 ] ; then
+    version=3.2.0
+else
+    version=$3
+fi
+
+build_deps="libc6-dev zlib1g-dev tcl-dev tk-dev libblas-dev liblapack-dev libcairo2-dev libpng12-dev libtiff4-dev libjpeg-dev libreadline-dev default-jre"
+urls="
+http://cran.rstudio.com/src/base/R-3/$pkg-$version.tar.gz"
```
Comment (bgruening) [inline R/build.sh]: unnecessary line break?

**Comment by bgruening** [comment]
Great! Thanks Dave!

File: R/build.sh (hunk 1 above)
Comment (natefoo) [inline R/build.sh]: `urls` is technically a list (insofar as a variable value separated by spaces or newlines is a list in Bourne), originally i wrote these as:

``` sh
urls="
http://example.org/example.tar.gz
"
```

To make that painfully obvious. It's not necessary to follow suit if there is only one url.

Comment (davebx) [inline R/build.sh]: Noted.

**Comment by natefoo** [comment]
:+1:, works for me with all the magical rpath stuff and included libs. Thanks @davebx

**Comment by natefoo** [comment]
https://depot.galaxyproject.org/package/linux/x86_64/R/R-3.2.0-Linux-x86_64.tar.gz

**Pull request merged**